internal/dag/httpproxy_processor.go

```diff
@@ -870,6 +870,13 @@ func (p *HTTPProxyProcessor) computeRoutes(
 				}
 			}
 
+			slowStartConfig, err := slowStartConfig(service.SlowStart)
+			if err != nil {
+				validCond.AddErrorf(contour_api_v1.ConditionTypeServiceError, "SlowStartInvalid",
+					"%s on slow start", err)
+				return nil
+			}
```

Comment:
it'd be nice to add some test cases to status_test.go to cover the error case here and make codecov stop complaining.

Reply:
Test added.  I also changed the printout a little so that the error we surface will not contain too much unnecessary details (which the error string directly from `strconv.ParseFloat()` had).